tests/node-firefly

```diff
@@ -27,11 +27,10 @@ cd openzeppelin-solidity
 git checkout "$open_zeppelin_commit"
 
 npm install
-npx truffle compile
 
 # Wait for kevm-vm to start up
 while ! netcat -z localhost "$kevm_vm_port"; do
     sleep 1
 done
 
-npx truffle test
+node_modules/.bin/truffle test
```

Comment:
Why these changes?

Reply:
So that we always use the version of truffle specified by the package rather than the latest version which may not be backwards compatible.